iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/TemporaryLimitsUtils.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public final class TemporaryLimitsUtils {
+
+    public static String getTmpLimitName(int acceptableDuration) {
+        if (acceptableDuration < 60) {
+            return "IT" + acceptableDuration + "s";
```

Comment:
ITXX is the way we name temporary limits specifically at RTE but it not a general naming rule. It is ok to name like this in test case but we should avoid having this kind of utility class in API.